History.md

```diff
@@ -69,6 +69,13 @@ N/A
   [Issue #10393](https://github.com/meteor/meteor/issues/10393)
   [PR #10414](https://github.com/meteor/meteor/pull/10414)
 
+* If your application is using Git for version control, the current Git
+  revision string will be exposed via `Meteor.gitRevision` while the app
```

Comment:
They're synonyms officially, though both seem to refer to the object rather than its string hash: https://git-scm.com/docs/gitglossary#gitglossary-aiddefrevisionarevision

How would you feel about `Meteor.gitCommitHash`?